WalletWasabi.Tests/UnitTests/WabiSabi/Integration/WabiSabiHttpApiIntegrationTests.cs

```diff
@@ -200,6 +204,8 @@ public async Task CoinJoinWithBlameRoundTestAsync(long[] amounts)
 					services.AddScoped<WabiSabiConfig>(s => new WabiSabiConfig
 					{
 						MaxInputCountByRound = 2 * inputCount,
+						StandardInputRegistrationTimeout = TimeSpan.FromSeconds(20),
```

Comment:
I don't know why 20, this is a quite real test and for that reason the timeouts are discovered empirically.

Reply:
yeah these tests have an implicit dependency on the performance of the test machine, too short and the range proof computations can cause timeouts leading to spurious failures, too long and it waits around pointlessly...

the root cause of this is the dependency on wallclock time in arena, and this is endemic in the tests all over the place (there are mainly tacit assumptions about things running faster than some wallclock time baked in, which are sometimes violated due to the parallelism).

the only way to actually address this is to abstract the scheduling of delays/time based transitions so that the tests themselves can control both `Arena` and the various clients' perception of time and decouple it from the actual clock